test/e2e/e2e_test.go

```diff
@@ -98,26 +98,24 @@ func TestWebhookInterceptor(t *testing.T) {
 	}
 
 	// Push a commit, which should trigger a webhook, which in turn should start a pipeline run.
-	filename := "ods.yml"
+	filename := "ods.yaml"
 	fileContent := `
-	version: 2022.2.0
-
-	branchToEnvironmentMapping:
-		- branch: master
-		  environment: dev
-
-	environments:
-		- name: dev
-		  stage: dev
-	pipeline:
-		tasks:
-		  - name: package-image
-			taskRef:
```

Comment:
@gerardcl I think the indentation is off here, which probably caused the pipeline to never start ... YAML strikes again! 😄